src/renderer/react/app/nodes/editNodeModal/EditNodeModal.jsx

```diff
@@ -1,47 +1,51 @@
 import { noop, mapValues } from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
-import { compose } from 'recompose';
+import { compose, withPropsOnChange } from 'recompose';
 import { createForm, createFormField } from 'rc-form';
-import { createSelector } from 'reselect';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
 
 import { listDescendants } from 'services/node';
 import { nodeShape } from 'react/shapes/node';
 import { formShape } from 'react/shapes/form';
+import Button from 'react/components/general/button/Button';
+import globalMessages from 'config/global.messages';
+import { getNodesMap } from 'redux/stores/nodes/selector';
 
-/**
- * Maps props to a rc-form friendly default value.
- *
- * @param {object} props - Props.
- * @returns {object} - Values.
- */
-const defaultValueMapper
-  = props => mapValues(props.node || {}, value => createFormField({ value }));
+const enhancer = compose(
+  connect(state => ({
+    nodesMap: getNodesMap(state),
+  }), null),
```

Comment:
Setting null as second parameter does not work, you need to give `() => ({})` which is pretty annoying.

I was thinking of writing a proxy like that
```js
import { connect } from 'react-redux';

const emptyObject = {};
const defaultMapDispatchToProps = () => emptyObject;

export default connect(
  mapStateToProps,
  mapDispatchToProps = defaultMapDispatchToProps,
  mergeProps = null,
) => connect(mapStateToProps, mapDispatchToProps, mergeProps);
```

This way we can just import it instead of importing connect + the empty `mapDispatchToProps` everytime.